src/sass/messaging/partials/_messaging-app.scss

```diff
@@ -9,9 +9,15 @@
 
 
 #messaging-app-header {
-  @media (max-width: $medium-screen) {
+  @include media-maxwidth($medium-screen) {
```

Comment:
Plenty of places where we should replace `@media` queries with the mixin, but that should be tackled separately.